vignettes/config_yml.Rmd

```diff
@@ -0,0 +1,479 @@
+
+```yaml
+    dir_abcd: "~/Desktop/test/abcd"
+```
+
+#### dir_raw
+
+`dir_raw`{.yaml} is a path to a directory that contains raw loanbook data to be used. It must be a single string/character value, and it must refer to a valid, accessible, local directory. As an example:
+
+```yaml
+    dir_raw: "~/Desktop/test/raw"
+```
+
+#### dir_matched
+
+`dir_matched`{.yaml} is a path to a directory that contains matched loanbook data to be used. It must be a single string/character value, and it must refer to a valid, accessible, local directory. As an example:
+
+```yaml
+    dir_matched: "~/Desktop/test/matched"
+```
+
+#### dir_output
+
+`dir_output`{.yaml} is a path to a directory where the outputs should be saved. It must be a single string/character value, and it must refer to a valid, accessible, local directory. As an example:
+
+```yaml
+    dir_output: "~/Desktop/test/output"
+```
+
+## file_names:
+
+The `file_names`{.yaml} section contains options to define the file names of locally accessible files found in the directories defined in the `directories`{.yaml} section. The directories and file names are defined separately to allow for flexibility in where and how your input and output files are stored. A full example `file_names`{.yaml} section might look like:
+
+```yaml
+  file_names:
+    filename_scenario_tms: "scenarios_2022_p4b.csv"
+    filename_scenario_sda: "scenarios_2022_ei_p4b.csv"
+    filename_abcd: "2023-02-17_AI_RMI_PACTA for Banks Free dataset_EO_2022Q4.xlsx"
+    sheet_abcd: "Company Indicators - PACTA Comp"
+```
+
+#### filename_scenario_tms
+
+`filename_scenario_tms`{.yaml} is the filename of the file that contains production based scenario data. It must be a single string/character value, and it must refer to a valid, accessible, local file. As an example:
+
+```yaml
+    filename_scenario_tms: "scenarios_2022_p4b.csv
+```
+
+#### filename_scenario_sda
+
+`filename_scenario_sda`{.yaml} is the filename of the file that contains emission intensity based scenario data. It must be a single string/character value, and it must refer to a valid, accessible, local file. As an example:
+
+```yaml
+    filename_scenario_sda: "scenarios_2022_ei_p4b.csv"
+```
+
+#### filename_abcd
+
+`filename_abcd`{.yaml} is the filename of the file that contains asset based company production data. It must be a single string/character value, and it must refer to a valid, accessible, local file. As an example:
+
+```yaml
+    filename_abcd: "2023-02-17_AI_RMI_PACTA for Banks Free dataset_EO_2022Q4.xlsx"
+```
+
+#### sheet_abcd
+
+`sheet_abcd`{.yaml} is the name of the sheet that contains asset based company production data in the file defined by `filename_abcd`{.yaml} and stored in the directory defined by `dir_abcd`{.yaml}. It must be a single string/character value, and it must refer to a valid, accessible, sheet name in the appropriate file. As an example:
```

Comment:
> also: since this implies that we are looking for a sheet in an xlsx file, do we maybe want to highlight expectations of file types?

mnore so than this already does?
https://github.com/RMI-PACTA/workflow.multi.loanbook/blob/3bac0a2a5c066a44b2fa3fcefeeb264f569c2041/vignettes/config_yml.Rmd#L244-L250

or you mean specifically in the description of `sheet_abcd`?